docs/device-emulation.md

```diff
@@ -30,31 +30,63 @@ Because we do not want to modify the daemon, we think it makes sense to *load* a
 state over D-Bus. Each phase can be controlled, which makes it easy to view, and edit, the recorded
 emulation data.
 
-For instance, calling `fwupdmgr emulation-tag` would ask the end user to choose a device to start
+## Tell the daemon to record a hotplug device
+
+If the device you will be recording from is *hotpluggable* you will use the **emulation-tag** feature for the daemon.
+Calling `fwupdmgr emulation-tag` would ask the end user to choose a device to start
 *recording* so that subsequent re-plugs are available to save.
 As the device state may not be persistent we save the device-should-be-recorded metadata in the
 pending database like we would do for a successful firmware update.
 
-To demo this, something like this could be done:
+Here is how a hotplugged device would be recorded from:
 
-    # connect ColorHug2
+    # connect ColorHug2 and use the device ID to tag it
     fwupdmgr emulation-tag b0a78eb71f4eeea7df8fb114522556ba8ce22074
     # or, using the GUID
     # fwupdmgr emulation-tag 2082b5e0-7a64-478a-b1b2-e3404fab6dad
     # remove and re-insert ColorHug2
     fwupdmgr get-devices --filter emulation-tag
+
+## Tell the daemon to record a persistent device
+
+Persistent devices are coldplugged or physically not removable from the system.  In this case you must
+tell the daemon by a list of device IDs to start recording data immediately at startup by specifying in the config file.
+
+    fwupdmgr modify-config fwupd EmulatedDevices b0a78eb71f4eeea7df8fb114522556ba8ce22074
```

Comment:
do we need this at all now that `fwupdmgr emulation-tag` is writing to the config file?

Reply:
Fixed it